src/vocgui/views/word_viewset.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import viewsets
+from rest_framework.authentication import SessionAuthentication, BasicAuthentication
+from rest_framework.permissions import IsAuthenticated
+from vocgui.models import Document
+from vocgui.serializers import DocumentSerializer
+
+
+class WordViewSet(viewsets.ModelViewSet):
+    """
+    Retrieve the list of all default words/documents or get a single record by appending the id of the requested word.
+    """
+
+    serializer_class = DocumentSerializer
+    http_method_names = ["get"]
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
+    permission_classes = [IsAuthenticated]
```

Comment:
Users in the app are generally not authenticated :)

Reply:
Ah ok, I think I incorrectly copied + pasted this from the [DocumentByIdViewSet](https://github.com/digitalfabrik/lunes-cms/blob/5edf4033cf403e856c464ab688ed41435eba383c/src/vocgui/views/document_by_id_viewset.py#L18) endpoint... is this a mistake there as well?

Reply:
No in DocumentByIdViewSet it's correct :-)